roles/rook/tasks/wait-for-cluster-deleted.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Wait 30m until CephCluster is Deleted
+  kubernetes.core.k8s_info:
```

Comment:
Missing in osism/zuul-jobs, I will add this there.

Reply:
:+1: